build/ublue-os-just/main.just

```diff
@@ -66,3 +66,8 @@ update:
 fixscreenshare:
   cp /usr/share/applications/org.kde.xwaylandvideobridge.desktop $HOME/.config/autostart/
   
+# Add user to libvirt group after layering libvirt packages
+usermod-libvirt:
```

Comment:
Can't we ship polkit rules that do the same thing?  

Is it better adding the user to the libvirt group or adding the necessary polkit rules?

Reply:
That would be way better than my one off approach. I am hoping that someone can fill in the gaps between mutable and immutable. In the mean time, using just gives a temporary way to get that done.